### ISSUE TITLE
Check for keyboard accessibility

### ISSUE THREAD

**Issue by mary-ext** [buttondown/roadmap#3580]
Tracking issue for checking keyboard accessibility, if I see any I'll point it out here before filing individual ones as necessary.

```[tasklist]
### Tasks
- [ ] Log out button on the sidebar can't be focused
- [x] Search button on the sidebar can't be focused (I think we should, just to have another way that isn't with Ctrl+K)
- [x] Checkboxes don't have focus ring, notable examples include table selections
- [x] Color picker doesn't have focus ring
- [x] Can't focus on color picker presets
- [x] Action button ring focus might not be very obvious (we should probably give this an outline)
- [x] Can't copy JSON viewer contents to clipboard
```
